app/src/main/java/com/github/h3lp3rs/h3lp/notification/EmergencyListener.kt

```diff
@@ -1,66 +1,112 @@
 package com.github.h3lp3rs.h3lp.notification
 
+import LocationHelper
 import android.content.Intent
 import android.os.Bundle
 import com.github.h3lp3rs.h3lp.*
 import com.github.h3lp3rs.h3lp.database.Databases
+import com.github.h3lp3rs.h3lp.database.Databases.*
 import com.github.h3lp3rs.h3lp.database.Databases.Companion.databaseOf
 import com.github.h3lp3rs.h3lp.dataclasses.EmergencyInformation
 import com.github.h3lp3rs.h3lp.dataclasses.HelperSkills
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+import com.github.h3lp3rs.h3lp.notification.NotificationService.Companion.createNotificationChannel
+import com.github.h3lp3rs.h3lp.notification.NotificationService.Companion.sendIntentNotification
 import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.globalContext
 import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.*
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
 
 object EmergencyListener {
 
+    // Maximum helpee-helper distance in meters
+    private const val MAX_DISTANCE = 5000
```

Comment:
That would be nice that the user can choose this parameters in settings.

This could be part of a new user story,

Reply:
Yes, thought of the same thing. Could be set in the preferences.